packages/pf-codemods/index.js

```diff
@@ -55,67 +63,84 @@ async function printResults(eslint, results, format) {
         return eslint.getRulesMetaForResults(results);
       }
       return rulesMeta;
-    }
+    },
   });
   console.log(output);
 
   return true;
 }
 
+function ruleVersion(options) {
+  if (options.v4) {
+    return [...ruleVersionMapping["v5"], ...ruleVersionMapping["v6"]];
+  } else if (options.v6) {
+    return [...ruleVersionMapping["v4"], ...ruleVersionMapping["v5"]];
+  } else {
+    return [...ruleVersionMapping["v4"], ...ruleVersionMapping["v6"]];
+  }
+}
```

Comment:
Just to confirm, the point of this is to return the rules thare *aren't* for the version specified in options? So `v4` in options means we want this to return the v5 and v6 rules so that we can remove them?

If so then a slight nit would be the function name; Reading this at first I thought this would be an array of the rules we'd want to run, but further down looks like these are the ones we're excluding.

Reply:
Yeah that was how I thought it worked initially before as well, I'll see what I can come up with for better name.